msg/DisplayRobotState.msg

```diff
@@ -4,5 +4,6 @@ RobotState state
 # Optionally, various links can be highlighted
 ObjectColor[] highlight_links
 
-# Optionally, hide the robot state (in rviz) if true
-bool hide
+# Optionally, do not display this state in visualizations
+# and hide previously sent states
+bool hide_display
```

Comment:
Actually, I changed back to `hide` intentionally (but I should have commented).
This variable never stands on its own, but it's always used with a prefix like `display_robot_state.hide`. In this context, I think the short name is sufficient.

Reply:
makes sense and I agree.